src/test/librbd/test_librbd.cc

```diff
@@ -2808,7 +2808,7 @@ TEST_F(TestLibRBD, ZeroLengthDiscard)
   ASSERT_EQ(0, rbd_open(ioctx, name.c_str(), &image, NULL));
 
   const char *data = "blah";
-  char read_data[strlen(data)];
+  char read_data[strlen(data)+1];
```

Comment:
this is a false alarm. could use
```c++
const char[] data = "blah";
```
and replace `strlen(data)` with `sizeof(data)` instead.

but i am also good with this change for less code churn.